app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Command can take only 2 arguments")
+    cp_command, file, new_file = command.split()
+    if cp_command != "cp":
+        raise ValueError("Command not found")
```

Comment:
The error message 'Command not found' is not very informative. It would be better to specify what command you are talking about. For example: 'cp command not found'.